client/src/components/Allocations/Allocations.scss

```diff
@@ -13,109 +13,63 @@
 /*
     Components
 */
-.btn-link {
-    color: #9d85ef;
-    padding-left: 0;
-}
```

Comment:
**Why?**

- [A Link is used, now](https://github.com/TACC/Frontera-Portal/pull/236/files#r515382252).
- This style has been moved to `index.css`, because [other components rely on it](https://github.com/TACC/Frontera-Portal/pull/236#issuecomment-721198861).